src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolumegroup-form/cephfs-subvolumegroup-form.component.html

```diff
@@ -27,6 +27,9 @@
             <span class="invalid-feedback"
                   *ngIf="subvolumegroupForm.showError('subvolumegroupName', formDir, 'notUnique')"
                   i18n>The subvolume group already exists.</span>
+            <span *ngIf="subvolumegroupForm.showError('subvolumegroupName', formDir, 'pattern')"
+                  class="invalid-feedback"
+                  i18n>Subvolume name can only contain letters, numbers, '.', '-' or '_'</span>
```

Comment:
by letters you mean Alphabets?

Reply:
yup

Reply:
Are we using it commonly in dashboard the word letters?

Reply:
Yeah, the `letters` is something which is commonly used everywhere in the dashboard.